site/content/entry/citra-progress-report-2017-p1.md

```diff
@@ -0,0 +1,33 @@
++++
+date = "2017-04-04T20:00:00-04:00"
+title = "Citra Progress Report - 2017 P1"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 00
++++
+
+It's a bit hard to believe, but Citra is already [three years old](https://github.com/citra-emu/citra/commit/7a7917df27cfe5c342bba6c15b57d389fce585bd), and in these three years we've gone from an emulator that could barely run homebrew, to one that can run many commercially available games at playable speeds! It's all thanks to the hard work of volunteers working to improve the emulator at a pace we could've only imagined back in the spring of 2014, and it seems 2017 shows no signs of slowing down. We've had a lot of improvements so far with accuracy and even more 3DS features, so let's take a look.
+
+## [Gamepad Support](https://github.com/citra-emu/citra/pull/2497) by [wwylele](https://github.com/wwylele)
+
+A very often requested feature we've had on the forums and Discord server has been gamepad support, and [wwylele](https://github.com/wwylele) has finally given us just that; gamepad support, using SDL as the input backend. In order to do this, they have completely rewritten the input handling architecture, replacing it with a much cleaner interface that will allow adding different input methods much more easily in the future.
```

Comment:
> Perhaps the most requested feature we've had on both the forums and Discord server has been gamepad support, and wwylele has finally delivered!

I used that exact same phrasing in the draft, but changed it due to it implying that this was @wwylele's responsibility, and was slacking off.

> Using SDL as the input backend, Citra now supports a huge variety of gamepads, including XBox controllers and PS4 Controllers.

Will definitely add this, and maybe one more controller, due to the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(writing)).